libbeat/template/template.go

```diff
@@ -311,6 +311,12 @@ func buildIdxSettings(ver common.Version, userSettings common.MapStr) common.Map
 		},
 	}
 
+	// default_pipeline is only supported for ES version >= 6.5
+	version65, _ := common.NewVersion("6.5.0")
+	if config.DefaultPipelineEnabled && !ver.LessThan(version65) {
+		indexSettings.Put("default_pipeline", config.DefaultPipelineName)
+	}
```

Comment:
The pipeline itself requires the Geoip plugin, which is only bundled into ES since 6.7. Might be worth checking for that.

Reply:
Ah, makes sense - changing to `6.7.0`. I confirmed it works out of the box with that Elasticsearch version.